ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/recovery/jts/jca/coordinator/RecoveredServerTransaction.java

```diff
@@ -0,0 +1,487 @@
+/*
```

Comment:
I think though that I can't do what you ask because I need to extend the JCA version of the ServerTransaction